getting-started/README.md

```diff
@@ -0,0 +1,264 @@
+# Getting started developing for Optimism
+
+[![Discord](https://img.shields.io/discord/667044843901681675.svg?color=768AD4&label=discord&logo=https%3A%2F%2Fdiscordapp.com%2Fassets%2F8c9701b98ad4372b58f13fd9f65f966e.svg)](https://discord.com/channels/667044843901681675)
+[![Twitter Follow](https://img.shields.io/twitter/follow/optimismPBC.svg?label=optimismPBC&style=social)](https://twitter.com/optimismPBC)
+
+This tutorial teaches you the basics of Optimism development. 
+Because of [EVM equivalence](https://medium.com/ethereum-optimism/introducing-evm-equivalence-5c2021deb306) the differences are minor, but [they do exist](https://community.optimism.io/docs/developers/build/differences/#).
+
+## Optimism endpoint URL
+
+To access any Ethereum type network you need an endpoint. There are several ways to get one:
+
+1. For the limited number of transactions required for development, rather than QA or production, you can use [Optimism-provided endpoints](https://community.optimism.io/docs/useful-tools/networks/). 
+
+1. There is a number of service providers that provide Optimism endpoints, usually with a free tier for low usage.
+
+   * [Alchemy](https://www.alchemy.com/layer2/optimism)
+   * [Infura](https://infura.io/docs/ethereum#section/Choose-a-Network)
+   * [QuickNode](https://www.quicknode.com/chains/optimism)
+
+1. If you prefer to keep your decentralized application development private, you can also [run a local development node](https://community.optimism.io/docs/developers/build/dev-node/).
+
+
+### Network choice
+
+For development purposes we recommend you use either a local development node or [Optimistic Kovan](https://kovan-optimistic.etherscan.io/).
+That way you don't need to spend real money.
+If you need Kovan ETH for testing purposes, [you can use this faucet](https://faucet.paradigm.xyz/).
+
+
+### Connecting the development stack to Optimism
+
+The exact steps to do this depends on the development stack you're using.
```

Comment:
... below we outline some of the most popular development stacks in the EVM space.